recovery_behavior/include/recovery_behavior/back_up.hpp

```diff
@@ -0,0 +1,80 @@
+// Licensed under MIT. See LICENSE file. Copyright Carologistics.
+
```

Comment:
remove long license header, keep line 1

Reply:
fixed, pls refer to latest commit [dce5e3e](https://github.com/carologistics/robotino_navigation/pull/14/commits/dce5e3ee9f1a73e0995f00660a3e138e691fc48